### ISSUE TITLE
Standard combined transport profile number for swap bodies does not point toward a skos:ConceptScheme

### ISSUE THREAD

**Issue by remiColletGG** [Interoperable-data/ERA_vocabulary#20]
Hello there,

Unless I am mistaken, [https://data-interop.era.europa.eu/era-vocabulary/index-en.html#profileNumberSwapBodies](https://data-interop.era.europa.eu/era-vocabulary/index-en.html#profileNumberSwapBodies) should probably point toward its [swap bodies profile concept scheme](https://virtuoso.ecdp.tech.ec.europa.eu/describe/?url=http%3A%2F%2Fdata.europa.eu%2F949%2Fconcepts%2Fprofile-num-swap-bodies%2FProfileNumbersSwapBodies&sid=269535 ). Currently, it only mentions Appendix A-1.

**Comment by ocorcho** [comment]
We will check it and let you know in this issue. Thanks for reporting.

**Comment by ednaru** [comment]
We checked in the ontology. The property [profileNumberSwapBodies ](https://data-interop.era.europa.eu/era-vocabulary/index-en.html#profileNumberSwapBodies)has as its range skos:Concept. An annotation property inSkosConceptScheme is used to point to the specific Concept Scheme, currentky this annotation is not generated in the HTML documentation but it does appear in the ontology:
inSkosConceptScheme <http://data.europa.eu/949/concepts/profile-num-swap-bodies/ProfileNumbersSwapBodies>
In the future we hope to improve the HTML documentation in order to include several of these annotations.

**Comment by gatemezing** [comment]
Indeed, in the draft version that we are using for the discussion with the working groups here https://linkedvocabs.org/data/era-ontology/3.1.0/doc/index-en.html#profileNumberSwapBodies, the documentation shows better the annotation property. 
<img width="1064" alt="profileNumberSwapBodies" src="https://github.com/Interoperable-data/ERA_vocabulary/assets/922883/cb543ff7-32cf-4682-917f-4e8a9e58a212">
Yes, the next release with take care of this aspect.

**Comment by remiColletGG** [comment]
The _Rinf Property annotation_ section indeed makes it much more user-friendly. Thanks a lot !

**Comment by Interoperable-data** [comment]
Fixed in this revision (work in progress for final publication), same link https://linkedvocabs.org/data/era-ontology/3.1.0/doc/index-en.html#profileNumberSwapBodies
![widoco-skos-fixed](https://github.com/user-attachments/assets/e23562b2-40ac-447a-a7dd-9d28d4f8b995)
